x/emissions/keeper/inference_synthesis/weight.go

```diff
@@ -11,10 +13,19 @@ import (
 
 // Given the current set of inferers and forecasters in the palette, calculate their
 // weights using the current regrets
-func (p *SynthPalette) CalcWeightsGivenWorkers() (RegretInformedWeights, error) {
+func calcWeightsGivenWorkers(
+	logger log.Logger,
```

Comment:
Is it preferred to pass in the Logger than to pass the Context? I see merit to both cases but just to make sure we are aligned on this decision and aim to keep it as a new standard.

Using Logger is lighter and more explicit, and because we don't normally use advanced use of Context in keeper, then it is simpler. 
Using Context adds uniformity and flex through all cases, regardless you need sth else from the Context or not.

Reply:
no strong opinion, slight preference for the logger for the reasons you outline

note that in other parts of the codebase, the context is used for things unrelated to the logger (majority in functions that interact with the keeper which takes ctx as an argument), so I doubt that there are many cases we would find even if we looked for them where we could substitute a ctx for just the logger